CHANGELOG.md

```diff
@@ -23,14 +29,6 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   ([#1480](https://github.com/open-telemetry/opentelemetry-python/pull/1480))
 - Allow missing carrier headers to continue without raising AttributeError
   ([#1545](https://github.com/open-telemetry/opentelemetry-python/pull/1545))
-- Add urllib to opentelemetry-bootstrap target list
```

Comment:
where did this line go?

Reply:
This will be added to the `0.18b` release since the change does not pertain to any `rc1` package.